src/rpc/ngHandlers/AccountChannels.h

```diff
@@ -0,0 +1,130 @@
+//------------------------------------------------------------------------------
+/*
+    This file is part of clio: https://github.com/XRPLF/clio
+    Copyright (c) 2023, the clio developers.
+
+    Permission to use, copy, modify, and distribute this software for any
+    purpose with or without fee is hereby granted, provided that the above
+    copyright notice and this permission notice appear in all copies.
+
+    THE  SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+    WITH  REGARD  TO  THIS  SOFTWARE  INCLUDING  ALL  IMPLIED  WARRANTIES  OF
+    MERCHANTABILITY  AND  FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+    ANY  SPECIAL,  DIRECT,  INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+    WHATSOEVER  RESULTING  FROM  LOSS  OF USE, DATA OR PROFITS, WHETHER IN AN
+    ACTION  OF  CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+    OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+*/
+//==============================================================================
+
+#pragma once
+
+#include <backend/BackendInterface.h>
+#include <rpc/common/Types.h>
+#include <rpc/common/Validators.h>
+
+#include <boost/asio/spawn.hpp>
+
+#include <functional>
+#include <vector>
+
+namespace RPCng {
+class AccountChannelsHandler
+{
+public:
+    // type align with SField.h
+    struct ChannelResponse
+    {
+        std::string channelID;
+        std::string account;
+        std::string accountDestination;
+        std::string amount;
+        std::string balance;
+        std::optional<std::string> publicKey;
+        std::optional<std::string> publicKeyHex;
+        uint32_t settleDelay;
+        std::optional<uint32_t> expiration;
+        std::optional<uint32_t> cancelAfter;
+        std::optional<uint32_t> sourceTag;
+        std::optional<uint32_t> destinationTag;
+    };
+
+    struct HandlerOutput
+    {
+        std::vector<ChannelResponse> channels;
+        std::string account;
+        std::string ledgerHash;
+        uint32_t ledgerIndex;
+        // validated should be sent via framework
+        bool validated = true;
+        uint32_t limit;
+        std::optional<std::string> marker;
+    };
+
+    struct HandlerInput
+    {
+        std::string account;
+        std::optional<std::string> destinationAccount;
+        std::optional<std::string> ledgerHash;
+        std::optional<uint32_t> ledgerIndex;
+        uint32_t limit = 50;
+        std::optional<std::string> marker;
+    };
+
+    using Input = HandlerInput;
```

Comment:
Is there point calling it `HandlerInput` and then aliasing it here? why not just call the struct Input and Output from the start, just curious if there is a reason other than having the `using` aliases nicely together in a group?